ms-docs/JBoss EAP on Azure.MD

```diff
@@ -1,161 +1,164 @@
 
-- **Start JBoss EAP as a Standalone Server** - Following is the command to start EAP service in Standalone Mode.
+## Server configuration choice
+
+For deployment of RHEL VM, you can either choose between Pay-As-You-Go (PAYG) or Bring-Your-Own-Subscription (BYOS). If you choose the VM image from the [Azure Marketplace](https://azuremarketplace.microsoft.com) this defaults to a PAYG image.  If you have a RHEL OS image that you want to use, then you can deploy a BYOS type RHEL VM. Make sure your RHSM account has BYOS entitlement via Cloud Access. 
+
+In addition to providing functionality and APIs to its applications, JBoss EAP has powerful management capabilities. These management capabilities differ depending on which operating mode is used to start JBoss EAP. JBoss EAP is supported on RHEL, Windows Server, and Oracle Solaris. JBoss EAP offers a standalone server operating mode for managing discrete instances and a managed domain operating mode for managing groups of instances from a single control point. Note that JBoss EAP managed domains are not supported in Microsoft Azure since the High Availability (HA) feature is managed by Azure infrastructure services. The environment variable named *EAP_HOME* is used to denote the path to the JBoss EAP installation. 
+
+- **Start JBoss EAP as a Standalone Server** - the following command is how you start EAP service in Standalone Mode.
 
     `$EAP_HOME/bin/standalone.sh`
     
-    This startup script uses the EAP_HOME/bin/standalone.conf file, to set some default preferences, such as JVM options. Settings can be customized in this file. JBoss EAP uses the standalone.xml configuration file to start on Standalone mode by default, but can be started using a different mode. For details on the available standalone configuration files and how to use them, see the [Standalone Server Configuration Files](https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.2/html/configuration_guide/jboss_eap_management#standalone_server_configuration_files) section. To start JBoss EAP with a different configuration, use the --server-config argument. For example:
+    This startup script uses the EAP_HOME/bin/standalone.conf file, to set some default preferences, such as JVM options. Settings can be customized in this file. JBoss EAP uses the standalone.xml configuration file to start on Standalone mode by default, but can be started using a different mode. For details on the available standalone configuration files and how to use them, see the [Standalone Server Configuration Files](https://access.redhat.com/documentation/red_hat_jboss_enterprise_application_platform/7.2/html/configuration_guide/jboss_eap_management#standalone_server_configuration_files) section. To start JBoss EAP with a different configuration, use the --server-config argument. For example:

```

Comment:
The Link mentioned for **Standalone Server Configuration Files** is not accessible without en-us/
https://access.redhat.com/documentation/red_hat_jboss_enterprise_application_platform/7.2/html/configuration_guide/jboss_eap_management#standalone_server_configuration_files